proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,567 @@
+the Articles of Association. The Board implements the by-laws following
+core member approval by internet vote.
+
+## 10. Meeting Minutes
+
+All decisions by the Board and General Assembly are documented in an 
+electronically translatable format in German and available for members’ 
+perusal and for the examination of the resolutions by the registry court.
+
+## 11. Internet vote
+
+The Association may conduct votes via the Internet. An Internet vote is valid
+after an announcement via email or mandatory group chat for core members, and
+if a quorum of at least 1/2 of the core members participates. The voting period
+must be at least two weeks after announcement to ensure that members have
+sufficient time to cast their votes.
+
+## 12. Labour Agreements
+
+The Board regulates remuneration for employees in the by-laws.
+
+## 13. Financing of the Association
+
+1. The Association is funded through membership fees, public contributions,
+   donations, and other financial contributions.
+2. Membership fees are decided by the General Assembly.
+3. In case of a dissolution or liquidation of the association or termination of
+   its tax-privileged status all its assets will become property of
+   K Desktop Environment e.V., Berlin, register number VR31685 Amtsgericht
+   Berlin (Charlottenburg), which has to use them exclusively and
+   directly for non-for-profit purposes only according the aims of this articles.
+
+## 14. Data Protection in the Association
+
+1. In order to fulfill the purposes and tasks of the association, personal data
+   of the members are processed in compliance with the provisions of the EU
+   General Data Protection Regulation (GDPR) and the Federal Data Protection
+   Act (BDSG).
+2. The organs of the Association and other persons working for it are
+   prohibited from processing personal data and private messages, or
+   making it accessible to third parties, without authorization for purposes
+   other than the assigned task. This obligation also applies after the
+   above-mentioned persons have left the Association.
+3. The Association honors letter secrecy for all communications, in accordance
+   with Article 10 of the German Basic Law, and commits to
+   equivalent standards for international communications where applicable.
+4. The Association respects the privacy choices of users of the provided
+   web services and distributed software, allowing them to participate
+   anonymously or with full identification.
+5. The Association is committed to transparency and will not collect any user
+   information without their explicit knowledge and consent.
+6. In cases where the Association's services are provided by a third-party
+   provider requiring a separate user login, the data protection policies of
+   the third-party provider will apply.
+
+## 15. Inception of the Articles of Association
+
+These Articles of Association came into effect following a General Assembly
+decision.
```

Comment:
After the founding meeting, the sentence should be amended or supplemented as follows: “These articles of association come into force with the resolution of the general meeting of ***date***.”